pkg/controller/scylladbcluster/resource.go

```diff
@@ -1070,14 +1056,143 @@ func makeMirroredRemoteConfigMaps(sc *scyllav1alpha1.ScyllaDBCluster, dc *scylla
 		})
 	}
 
-	err := apimachineryutilerrors.NewAggregate(errs)
+	err = apimachineryutilerrors.NewAggregate(errs)
 	if err != nil {
 		return progressingConditions, requiredRemoteConfigMaps, fmt.Errorf("can't make mirrored ConfigMaps: %w", err)
 	}
 
 	return progressingConditions, requiredRemoteConfigMaps, nil
 }
 
+// getConfigMapsToMirrorForAllDCs collects the names of ConfigMaps to mirror for all Datacenters in the ScyllaDBCluster.
+func getConfigMapsAndSecretsToMirrorForAllDCs(sc *scyllav1alpha1.ScyllaDBCluster) ([]string, []string, error) {
```

Comment:
why have a common func for both secrets and configmaps? It sems like the flow would be more natural in both the implementation (no need to ignore returned values) and testing if they were separate?

Reply:
To avoid duplication of two long functions. Look at how getConfigMapsAndSecretsToMirrorForDC look like, it used to look like this but for just secrets, we would have to have a duplicated one, but collecting configmaps from the same spec paths.

Reply:
that's fair enough